qemu/tests/virtio_fs_share_data.py

```diff
@@ -211,6 +211,20 @@ def get_viofs_exe(session):
                 if md5_guest != md5_host:
                     test.fail('The md5 value of host is not same to guest.')
 
+                # during virtio fs is mounted, reboot vm
+                if params.get('reboot_guest', 'False'):
+                    error_context.context("Reboot guest and check virtiofs daemon.",
+                                          logging.info)
+                    cmd_ps_virtiofsd = params.get('cmd_ps_virtiofsd')
+                    ps_vfsd = process.system_output(cmd_ps_virtiofsd, shell=True)
+                    session = vm.reboot(session)
+                    ps_vfsd_reboot = process.system_output(cmd_ps_virtiofsd,
+                                                           shell=True)
+                    if ps_vfsd != ps_vfsd_reboot:
+                        test.fail("Virtiofs daemon is different before and after reboot.\n"
+                                  "Before reboot: %s\n"
+                                  "After reboot: %s\n", (ps_vfsd, ps_vfsd_reboot))
```

Comment:
This reboot guest and check virtiofs pid is not belong to the loop `for fs in params.objects("filesystems"):`
Should associate to guest object.